v3/plugins/ms-vscode/node-debug/1.35.2/meta.yaml

```diff
@@ -13,7 +13,7 @@ category: Debugger
 firstPublicationDate: '2019-06-20'
 spec:
   containers:
-  - image: "docker.io/eclipse/che-theia-endpoint-runtime:next"
+  - image: "docker.io/eclipse/che-remote-plugin-node:next"
```

Comment:
we should just remove these entirely -- node-debug is for supporting Node versions < 8. See https://github.com/eclipse/che-plugin-registry/pull/264